design/badges/standards.md

```diff
@@ -0,0 +1,26 @@
+# Badge standards for less problems
+Written by [@firefoxofficial](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/firefoxofficial.md)
```

Comment:
```suggestion

```

upcoming style guide says to not do attributions, it's not merged yet but let's get ahead of it